packages/platforms/ios/src/index.ts

```diff
@@ -1,8 +1,87 @@
-import { Profiler, ProfilerPollingOptions, ScreenRecorder } from "@perf-profiler/types";
+import { Measure, Profiler, ProfilerPollingOptions, ScreenRecorder } from "@perf-profiler/types";
+import { ChildProcess, exec } from "child_process";
+
+interface AppMonitorData {
+  Pid: number;
+  Name: string;
+  CPU: string;
+  Memory: string;
+  DiskReads: string;
+  DiskWrites: string;
+  Threads: number;
+  Time: string;
+}
+
+interface FPSData {
+  currentTime: string;
+  fps: number;
+}
+
+type DataTypes = "cpu" | "fps";
 
 export class IOSProfiler implements Profiler {
+  private measures: Record<string, Measure> = {};
+  private lastFPS: FPSData = { currentTime: "", fps: 0 };
+  private lastCpu: AppMonitorData = {
+    Pid: 0,
+    Name: "",
```

Comment:
bad practice to leave `""` as default value
prefer using `null` to explicitely set the value to missing